demo/server.js

```diff
@@ -19,7 +19,8 @@ initSqlJs().then(SQL => {
     db = new SQL.Database();
     sqlstr = "CREATE TABLE model (id INTEGER PRIMARY KEY, name TEXT, address TEXT, description TEXT, model_type TEXT, encoder TEXT, accuracy NUMBER);"
       + "CREATE TABLE data (transaction_hash TEXT PRIMARY KEY, text TEXT);"
-      + "CREATE INDEX index_address ON model(address);";
+      + "CREATE INDEX index_address ON model(address);"
+      + "CREATE TABLE time_accuracy ( transaction_hash TEXT,block_no INTEGER  , model_id INTEGER ,accuracy NUMBER , timestamp DATETIME DEFAULT CURRENT_TIMESTAMP ,FOREIGN KEY (model_id) REFERENCES model (id)); ";
```

Comment:
Let's use clearer names. So how about "block_number" instead of "block_no"?
How about just "accuracy" for the table name?
Also would you make the styling consistent with the rest of the command so that it's easier to read: no space after "(", no spaces before ",", just one space after ",".

Reply:
okay sure